packages/stark-ui/src/modules/app-logo/components/app-logo.component.scss

```diff
@@ -10,25 +10,12 @@
 .stark-app-logo {
   & i {
     display: block;
-    width: 60px;
-    height: 64px;
-    margin-top: 4px;
-    background-image: url(/assets/stark-ui/app-logo/angular.png); /* FIXME: add image for tablet */
+    width: 150px;
+    height: 60px;
+    margin-top: 0;
+    background-image: url(/assets-base/stark_logo_transparent.svg);
```

Comment:
This path is wrong, the right path should be the one you define in the `angular.json` file of the stark-core ;)

Basically, you need to add the "assets" entry in the `angular.json` file in `packages/stark-core` to add these logos so they will be copied into the app assets while building the app. See how is done in the `angular.json` file in `packages/stark-ui`